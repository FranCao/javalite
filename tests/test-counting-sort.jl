int[] countingSortArray = [0,0,0,0,0,0,0,0,0,0
		        ,0,0,0,0,0,0,0,0,0,0
		        ,0,0,0,0,0,0,0,0,0,0
		        ,0,0,0,0,0,0,0,0,0,0
		        ,0,0,0,0,0,0,0,0,0,0
		        ];
int countingSortArrayLength = 50;
int[] sortingArr = [32,17,31,5,26,13,47,31,9,3,24,41,43,12,11];
int length = 15;
int i = 0;
int j = 0;
   
for (i = 0 ; i < length ; i = i + 1) {
	countingSortArray[sortingArr[i]] = countingSortArray[sortingArr[i]] + 1;   	
}
   
for (i = 0 ; i < countingSortArrayLength ; i = i + 1) {
   if(countingSortArray[i] > 0){
   	for(j=0;j<countingSortArray[i];j = j+1){
   		print(i);
   	}
   }   	
}
