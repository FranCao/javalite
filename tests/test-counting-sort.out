3
5
9
11
12
13
17
24
26
31
31
32
41
43
47
